cmd/harbor/root/registry/list.go

```diff
@@ -23,10 +23,14 @@ func ListRegistryCommand() *cobra.Command {
 				log.Fatalf("failed to get projects list: %v", err)
 			}
 			FormatFlag := viper.GetString("output-format")
-			if FormatFlag != "" {
+			if FormatFlag == "json" {
 				utils.PrintPayloadInJSONFormat(registry)
 				return
 			}
+			if FormatFlag == "yaml" {
+				utils.PrintPayloadInYAMLFormat(registry)
+				return
+			}
```

Comment:
Produce an error message to user when using flags other than json or yaml. 

![bugInlisting](https://github.com/user-attachments/assets/32cca321-0585-4fba-a0b4-70e484965fd8)

Reply:
I have finalized the subcommand `list` for `project`, `registry`, and `user`. I would like to confirm that this is accurate before moving on to support the remaining commands.

Reply:
Should I use a generic function to unify the output in both YAML and JSON formats?